test/decl/protocol/special/coding/class_codable_simple_extension.swift

```diff
@@ -1,29 +1,33 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -verify-ignore-unknown
```

Comment:
Where is there an unknown location? (as opposed to a cross-file location)

Reply:
I'm currently re-building Swift now so I can't at the moment get at the exact unknown locations, but when `Codable` derived conformance fails, the unit tests pick up a few `<unknown>:0:0:` messages that don't actually appear during compilation. This has been an issue across all of these tests (hence why other files have `-verify-ignore-unknown`); I've only added it where necessary to get tests to run correctly.

Reply:
We should figure out where those locations are coming from, because they're bad UX.

Reply:
Sure. These never appear to anyone who's doing real compilation (they only appear if you do _just_ type checking, which is really only relevant to unit tests), but I'll investigate a bit further.